src/BIAnalyticsService/CreateDataAssets/CreateDataAssets.cs

```diff
@@ -0,0 +1,198 @@
+            episode = JsonSerializer.Deserialize<Episode>(episodeJson);
+        }
+
+        catch (Exception ex)
+        {
+            _logger.LogError("Issue when getting episode from db. \nException: {ex}", ex);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        string nhsNumber = "1111111112";
+
+        var baseParticipantUrl = Environment.GetEnvironmentVariable("GetParticipantUrl");
+        var participantUrl = $"{baseParticipantUrl}?nhs_number={nhsNumber}";
+        _logger.LogInformation("Requesting participant URL: {Url}",participantUrl);
+
+        Participant participant;
+
+        try
+        {
+            var participantResponse = await _httpRequestService.SendGet(participantUrl);
+
+            if (!participantResponse.IsSuccessStatusCode)
+            {
+                _logger.LogError($"Failed to retrieve participant data with NHS number {nhsNumber}. Status Code: {participantResponse.StatusCode}");
+                return req.CreateResponse(participantResponse.StatusCode);
+            }
+
+            var participantJson = await participantResponse.Content.ReadAsStringAsync();
+            _logger.LogInformation("Participant data retrieved");
+
+            participant = JsonSerializer.Deserialize<Participant>(participantJson);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError("Issue when getting participant from db. \nException: {ex}", ex);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        var (screeningEpisodeUrl, screeningProfileUrl) = GetConfigurationUrls();
+        if (string.IsNullOrEmpty(screeningEpisodeUrl) || string.IsNullOrEmpty(screeningProfileUrl))
+        {
+            _logger.LogError("One or both URLs are not configured. \nUrl:{screeningProfileUrl}\nUrl:{screeningEpisodeUrl}", participantUrl, screeningEpisodeUrl);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        try
+        {
+            await SendToCreateParticipantScreeningEpisodeAsync(episode, screeningEpisodeUrl);
+            await SendToCreateParticipantScreeningProfileAsync(participant, screeningProfileUrl);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Failed to create episode or profile.");
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        return req.CreateResponse(HttpStatusCode.OK);
+    }
+
+    private (string screeningEpisodeUrl, string screeningProfileUrl) GetConfigurationUrls()
+    {
+        return (Environment.GetEnvironmentVariable("CreateParticipantScreeningEpisodeUrl"), Environment.GetEnvironmentVariable("CreateParticipantScreeningProfileUrl"));
+    }
+
+    private async Task SendToCreateParticipantScreeningProfileAsync(Participant participant, string screeningProfileUrl)
+    {
+        if (participant != null)
```

Comment:
This null check can be removed. If participant is somehow null then the following code will throw an exception which will be caught in our catch block on line 104